src/Exporter.php

```diff
@@ -116,7 +116,7 @@ public function add_import_button(): void
         <script>
             jQuery(function(){
                 jQuery(".wrap .page-title-action")
-                    .after('<a href="admin.php?import=wordpress" class="page-title-action"><?php esc_html_e('Import', 'planet4-master-theme-backend'); ?></a>');
+                    .after('<a href="upload.php?page=media-picker" class="page-title-action"><?php esc_html_e('Import', 'planet4-master-theme-backend'); ?></a>');
```

Comment:
I think this button also shows up on page/post/action list, not only media library, maybe we could keep the wp import link for those ?

Reply:
No, I think in those cases the behavior is expected. It's the Media Library import that confuses the editors because they expect to import an image.